drivers/node/ssh/ssh.go

```diff
@@ -194,6 +194,17 @@ func (s *SSH) Init(nodeOpts node.InitOptions) error {
 	return nil
 }
 
+// Init initializes SSH node driver
+func (s *SSH) Init(nodeOpts node.InitOptions) error {
+	s.specDir = nodeOpts.SpecDir
+	return s.updateDriver()
+}
+
+// Refreshes the driver on a context switch
+func RefreshDriver(s *SSH) error {
```

Comment:
Why use RefreshDriver when we already have the updateDriver function defined in SSH struct? Instead of passing the s *SSH parameter to RefreshDriver, can't we just call s.updateDriver() when needed? And, is updateDriver really appropriate? How about verifyStorageDrivers or something else?

Reply:
I named it `RefreshDriver` cuz in the future, other drivers might also be used, which would require a function that is defined by the interface and not just in `ssh`, which should have a relevant (general) and not a specific name, as `updateDriver` and `verifyStorageDrivers` seem specific and not generic.

@mkoppal-px any comment?